bmq-sdk/src/main/java/com/bloomberg/bmq/QueueOptions.java

```diff
@@ -294,6 +304,30 @@ public Builder setSuspendsOnBadHostHealth(boolean suspendsOnBadHostHealth) {
             return this;
         }
 
+        /**
+         * Adds subscription parameters for a specified subscription. Update subscription parameters
+         * if subscription with current 'subscription.getId()' previously added.
+         *
+         * @param subscription subscription parameters
+         * @return Builder this object
+         */
+        public Builder addSubscription(Subscription subscription) {
+            this.subscriptions.add(subscription);
+            return this;
+        }
+
+        /**
+         * Remove subscription parameters.
+         *
+         * @param subscription subscription parameters
+         * @return Builder this object
+         */
+        public Builder removeSubscription(Subscription subscription) {
+            // todo check exception if not found
```

Comment:
We should either throw an exception or add a comment above saying something like  "... if they have been added before"